plugins/inputs/snmp/snmp.go

```diff
@@ -37,13 +37,9 @@ const sampleConfig = `
   ## SNMP version; can be 1, 2, or 3.
   # version = 2
 
-  ## Translator for OIDs
-  ## Valid values are "netsnmp" which calls snmptranslate and snmptable,
-  ## and "gosmi" which uses the gosmi library
-  # translator = "netsnmp"
-
   ## Path to mib files
-  ## Only used with GoSMI translator
+  ## Used by the gosmi translator.
+  ## To add paths when translating with netsnmp, use the MIBDIRS environment variable
```

Comment:
I think we could even adapt that the netsnmp translator adds these configured paths to the commands as well, then it's transparant and only one place to configure.

Reply:
I thought the same thing. We could append to the MIBDIRS environment variable when calling netsnmp. It's too much of a risk to do right now while we're changing the default back to netsnmp.

Reply:
Yes, could be added in another PR indeed.